.github/workflows/ci-cd.yml

```diff
@@ -189,7 +189,7 @@ jobs:
             pip-ci-${{ runner.os }}-${{ matrix.pyver }}-${{ matrix.no-extensions }}-
     - name: Update pip, wheel, setuptools, build, twine
       run: |
-        python -m pip install -U pip wheel setuptools build twine
+        python -m pip install -U pip wheel setuptools build twine cffi==0.17.0rc1
```

Comment:
This seems redundant as the following step would do the same.

Reply:
Yeah, but `pip` (I think) has a dependency on cffi via cryptography

Reply:
Pip vendors all its deps so this would do nothing. Besides, all pip's deps are pure-python.